<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="1">
     <widget class="QSpinBox" name="R3I_SB">
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Initial resource quantity</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Number of producer</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Rules</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Initial resource quantity</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0" colspan="5">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="15" column="1">
     <widget class="QSpinBox" name="R5N_SB">
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Initial resource quantity</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QSpinBox" name="R3N_SB">
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Ressource 2 : Silver</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="2">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Ressource 5 : Seeds</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Ressource 3 : Oil</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="R1I_SB">
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Ressource 4 : Gaz</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_1">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Producer</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Ressource 1 : Gold</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Number of producer</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>Number of producer</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QCheckBox" name="COORD_CB">
      <property name="text">
       <string>Coordinated actions</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Number of producer</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0" colspan="5">
     <widget class="QPushButton" name="launch_PB">
      <property name="text">
       <string>Launch Simulation</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Initial resource quantity</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>Cooperative</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Initial resource quantity</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QSpinBox" name="R4N_SB">
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="R2N_SB">
      <property name="maximum">
       <number>5</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QSpinBox" name="R4I_SB">
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QSpinBox" name="R5I_SB">
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Number of producer</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="R2I_SB">
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>2000</number>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="R1N_SB">
      <property name="maximum">
       <number>5</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QSpinBox" name="AGENTC_SB">
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Agent</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>Individualistic</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QSpinBox" name="AGENTI_SB">
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="value">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>Humans</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QSpinBox" name="AGENTH_SB">
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QCheckBox" name="EXHAUST_CB">
      <property name="text">
       <string>Exhaustible resource</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QCheckBox" name="OBS_CB">
      <property name="text">
       <string>Allow observation</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QCheckBox" name="ROBBERY_CB">
      <property name="text">
       <string>Robbery</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QSpinBox" name="MAXGET_SB">
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>500</number>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Max resource to get</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QSpinBox" name="ENDCOND_SB">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100000000</number>
      </property>
      <property name="value">
       <number>10000</number>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>End Condition</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>launch_PB</tabstop>
  <tabstop>R1I_SB</tabstop>
  <tabstop>R1N_SB</tabstop>
  <tabstop>R2I_SB</tabstop>
  <tabstop>R2N_SB</tabstop>
  <tabstop>R3I_SB</tabstop>
  <tabstop>R3N_SB</tabstop>
  <tabstop>R4I_SB</tabstop>
  <tabstop>R4N_SB</tabstop>
  <tabstop>R5I_SB</tabstop>
  <tabstop>R5N_SB</tabstop>
  <tabstop>AGENTC_SB</tabstop>
  <tabstop>AGENTI_SB</tabstop>
  <tabstop>AGENTH_SB</tabstop>
  <tabstop>COORD_CB</tabstop>
  <tabstop>EXHAUST_CB</tabstop>
  <tabstop>OBS_CB</tabstop>
  <tabstop>ROBBERY_CB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
